components/Dashboard.js

```diff
@@ -91,14 +92,14 @@ const DashboardCard = styled.article`
     'weight weight height height';
   gap: 0.2em 0.2em;
   padding: 0.5em 1em;
-  border: 1px solid var(--text-primary);
+  border: 2px solid var(--text-primary);
   color: var(--text-primary);
   border-radius: 1em;
   box-shadow: var(--shadow-elevation);
   height: auto;
   max-height: 12.5em;
   width: 95%;
-  max-width: 32em;
+  max-width: 500px
```

Comment:
Just because I'm curious: Why did you switch to an absolute value?

Reply:
it's a matter of styling. I never want the dashboard to be wider than a  specific, fixed number of pixels. it shouldn't change even if you increase or decrease the font-size.